docs/organizations/roles-and-permissions-for-organizations.md

```diff
@@ -88,6 +90,14 @@ The table below maps the GitHub Cloud and GitHub Enterprise roles to the corresp
       <td class="yes">Yes</td>
       <td class="yes">Yes</td>
     </tr>
+    <tr>
+      <td>Manage gate policies</td>
+      <td>No</td>
+      <td colspan="2">No</td>
+      <td colspan="2">No</td>
+      <td>No</td>
+      <td class="yes">Yes</td>
+    </tr>
```

Comment:
Here, since the type of management capabilities that this new feature introduces is quite analogous to the coding standards and the permissions are also the same, I'd include them together in the same table line.

Reply:
Yes, that makes all the sense. 👍

Reply:
Updated!